pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SinksBase.java

```diff
@@ -167,14 +290,18 @@ public SinkConfig getSinkInfo(final @PathParam("tenant") String tenant,
             response = SinkStatus.class
     )
     @ApiResponses(value = {
-            @ApiResponse(code = 400, message = "Invalid request"),
-            @ApiResponse(code = 403, message = "The requester doesn't have admin permissions"),
-            @ApiResponse(code = 404, message = "The sink doesn't exist")
+            @ApiResponse(code = 400, message = "Invalid get status request"),
+            @ApiResponse(code = 401, message = "The client is not authorized to perform this operation"),
+            @ApiResponse(code = 404, message = "The sink doesn't exist"),
+            @ApiResponse(code = 503, message = "Function worker service is now initializing. Please try again later."),
```

Comment:
@yjshen can you add the 503 response to every endpoint since that can be returned from every sink endpoint

Reply:
done.